modules/apps/journal/journal-web/src/main/resources/META-INF/resources/article/basic_info.jsp

```diff
@@ -93,7 +93,7 @@ DDMStructure ddmStructure = journalEditArticleDisplayContext.getDDMStructure();
 		availableLocales="<%= journalEditArticleDisplayContext.getAvailableLocales() %>"
 		cssClass="form-control"
 		defaultLanguageId="<%= journalEditArticleDisplayContext.getDefaultArticleLanguageId() %>"
-		editorName="ckeditor"
+		editorName='<%= PropsUtil.get("editor.wysiwyg.portal-impl.portlet.ddm.text_html.ftl") %>'
```

Comment:
This is not our call to make. On top of that, this is not `ddm.text_html` so I don't think it's expected that it changes with that property

Reply:
@jbalsas thanks, I understand **we** shouldn't change it, but it did change something. I'm going to take a deeper look

Reply:
It is up to the echo team if they want basic info option to have a customizable editor or if it's hardcoded to `ckeditor`. If they do want a customizable editor, we would would need to implement beckend for it, for example add new property
```
editor.wysiwyg.portal-web.docroot.html.portlet.journal.basic_info.jsp=ckeditor
```
etc.